scaper/core.py

```diff
@@ -1738,40 +1742,50 @@ def _generate_audio(self, audio_path, ann, reverb=None,
                             tmpfiles_internal.append(
                                 tempfile.NamedTemporaryFile(
                                     suffix='.wav', delete=False))
-                            # synthesize concatenated/trimmed background
-                            cmb.build(
-                                [e.value['source_file']] * ntiles,
-                                tmpfiles_internal[-1].name, 'concatenate')
+                            # read in background off disk
+                            event_audio_array, event_sample_rate = soundfile.read(
```

Comment:
lets rename `event_audio_array` to `event_audio` (everywhere) to avoid being too close to `event_audio_arrays`

Reply:
Done.